src/rupali_umagol/Assignment79.java

```diff
@@ -0,0 +1,32 @@
+package rupali_umagol;
+
+public class Assignment79 {
+
+	void patternOneToSix() {
+		for (int i = 1; i <= 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible